asreview/webapp/src/ProjectComponents/AnalyticsComponents/NumberCard.js

```diff
@@ -18,7 +18,9 @@ export default function NumberCard(props) {
       props.progressQuery.isSuccess
     );
   };
-
+  const hasPriorKnowledge = () => {
+    return showNumber() && props.progressQuery.data["n_included"] > 0;
+  };
```

Comment:
This might resolve the problem partially. 

When uploading a dataset and including/excluding a couple of records, this number will still display. I think we should also disable this. The only solution I see at the moment is to extend https://github.com/asreview/asreview/blob/99bccf2ac8b9319597f60a3ba19189c3121991a3/asreview/webapp/api/projects.py#L1146 with a key-value for irrelevant since last relevant without priors.